community-operators/jaeger/jaeger.v1.11.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,303 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: jaeger-operator.v1.11.0
+  namespace: placeholder
+  annotations:
+    categories: "Logging & Tracing"
+    certified: "false"
+    containerImage: docker.io/jaegertracing/jaeger-operator:1.11.0
+    createdAt: "2019-03-26T12:00:00Z"
+    capabilities: Seamless Upgrades
+    support: Jaeger
+    description: Provides tracing, monitoring and troubleshooting microservices-based distributed systems
+    repository: https://github.com/jaegertracing/jaeger-operator
+    alm-examples: |-
```

Comment:
Should there be examples for the other CRDs as well?

Reply:
There is only actually 1 CRD - but just two versions - this version (1.11.0) of the operator only uses version v1 of the crd.